theme/templates/newbase.html

```diff
@@ -42,6 +42,9 @@
         .dropdown-toggle:after {
             display: none;
         }
+        .navbar-collapse {
+            background: white;
+        }
         .navbar-fixed-top {
```

Comment:
CSS, includes, extends and other template imports need to be thoroughly cleaned up across hydroshare. For now, a workaround overriding whatever conflict or unintended styling has cascaded to this point.